wandb/sdk/wandb_setup.py

```diff
@@ -62,26 +62,16 @@ def exception(self, msg: str, *args: Any, **kwargs: Any) -> None:
     def log(self, level: str, msg: str, *args: Any, **kwargs: Any) -> None:
         self._log.append((level, msg, args, kwargs))
 
-    def _flush(self) -> None:
-        assert self is not logger
-        assert logger is not None
+    def _flush(self, new_logger: Logger) -> None:
```

Comment:
probably can replace this EarlyLogger with custom handler and use logging.logger?

Reply:
yes, but for this PR, I just want to update types